OpenKeychain/src/main/AndroidManifest.xml

```diff
@@ -800,6 +800,20 @@
                 android:resource="@xml/custom_pgp_contacts_structure" />
         </service>
 
+        <service
+            android:name=".service.KeyserverSyncAdapterService"
+            android:exported="true"
```

Comment:
The guide at https://developer.android.com/training/sync-adapters/creating-sync-adapter.html said it was necessary so the system (the Android SyncAdapter framework, I assume) can access it?